great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -208,11 +216,13 @@ def _init_domain_builder(
     def _init_parameter_builders(
         parameter_builder_configs: Optional[List[CommentedMap]] = None,
         data_context: Optional["DataContext"] = None,  # noqa: F821
-    ) -> List[ParameterBuilder]:
+    ) -> Optional[List[ParameterBuilder]]:
         if parameter_builder_configs is None:
-            parameter_builder_configs = []
+            return None
```

Comment:
Sneaking in change per discussion with @alexsherstinsky  - if our schema and configs have this attr as being optional, we should reflect that in our init through an optional return value (and not an empty list)